cmd/nerdctl/build_test.go

```diff
@@ -48,6 +48,104 @@ CMD ["echo", "nerdctl-build-test-string"]
 	base.Cmd("run", "--rm", imageName).AssertOutExactly("nerdctl-build-test-string\n")
 }
 
+func TestBuildBaseImage(t *testing.T) {
+	if os.Getenv("TEST_BUILDKIT_WORKER") != "containerd" {
+		t.Skip()
```

Comment:
Needs explanation

Reply:
Now we don't need this envvar because we always run tests with containerd worker.